src/header.iced

```diff
@@ -0,0 +1,90 @@
+{prng} = require('crypto')
+{createHash} = require('crypto')
+msgpack = require('purepack')
+crypto = require('keybase-nacl')
+nonce = require('./nonce.iced')
+
+compute_mac_key = (encryptor, header_hash, pubkey) ->
+  zero_bytes = Buffer.alloc(32)
+  mac_box = encryptor.encrypt({plaintext : zero_bytes, nonce : nonce.nonceForMACKeyBox(header_hash), pubkey})
+  return mac_box.slice(-32)
```

Comment:
It might make sense to use the NaCl-provided constants for offsets like this, to make it more clear why we chose them? (I this case I think it would be `crypto_onetimeauth_BYTES`.)